lib/kitchen/directions/bake_chapter_summary.rb

```diff
@@ -30,12 +30,16 @@ def bake(chapter:, metadata_source:, uuid_prefix: '.', klass: 'summary')
             summary.first("[data-type='title']")&.trash # get rid of old title if exists
             summary_title = page.title.copy
             summary_title.name = 'h3'
-            summary_title.replace_children(with: <<~HTML
-              <span class="os-number">#{chapter.count_in(:book)}.#{page.count_in(:chapter)}</span>
-              <span class="os-divider"> </span>
-              <span class="os-text" data-type="" itemprop="">#{summary_title.children}</span>
-            HTML
-            )
+
+            unless summary_title.children.search('span.os-number').present?
```

Comment:
Is this to avoid double-baking something?

Reply:
Yes, Microbio already has the number and divider on the assembled version so it was adding the number twice